docs/AskSUSI.md

```diff
@@ -35,6 +35,26 @@ _example:_ Please explain insomnia.
 
 - Who tweeted most about `<query-term>` ? *OR* The top tweeter about `<query-term>`
 - How many times was `<query-term>` tweeted in the last 24 hours ?
+- How many tweets did `<query-username>` post in May 2016 ?
```

Comment:
@shivenmian I think this `May 2016` could be variable. Is it ?
If yes, pls ensure that it is mentioned as a placeholder, and a specific example be mentioned underneath following the pattern in the doc